scaffolds/Lite/package.json

```diff
@@ -5,50 +5,38 @@
   "engines": {
     "node": ">=8.0.0"
   },
-  "files": [
-    "src/",
-    "build/",
-    "public/",
-    "tests/",
-    "_gitignore",
-    "_editorconfig",
-    "_eslintignore",
-    "_eslintrc.js",
-    "_stylelintignore",
-    "_stylelintrc.js",
-    "ice.config.js"
-  ],
   "scripts": {
-    "start": "ice-scripts dev",
-    "build": "ice-scripts build",
-    "prepublishOnly": "npm run build & npm run screenshot",
-    "lint": "npm run eslint && npm run stylelint",
-    "eslint": "eslint --cache --ext .js,.jsx ./",
-    "stylelint": "stylelint ./**/*.scss",
-    "screenshot": "screenshot -l"
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "lint": "node_modules/eslint/bin/eslint.js src",
+    "test": "react-scripts test"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
   },
   "dependencies": {
     "@alifd/next": "^1.10.1",
-    "@icedesign/theme": "^1.x",
+    "node-sass": "^4.12.0",
```

Comment:
Don't need node-sass and `@alife/next`

Reply:
The `PageLoading` component [depends](https://github.com/ice-lab/cra-materials/blob/master/scaffolds/Lite/src/components/PageLoading/index.jsx#L2) on `@alifd/next` and [router.jsx](https://github.com/ice-lab/cra-materials/blob/master/scaffolds/Lite/src/router.jsx#L42) uses `PageLoading` component.

Reply:
div is ok, just simple loading